tests/encrypted_lvm.spec.ts

```diff
@@ -14,37 +15,69 @@ test.describe('The main page', () => {
     });
 
     test('Installation test with encrypted lvm file system', async ({ page }) => {
+        // optional actions done on the page
```

Comment:
s/optional actions done on the page/possible first screens the user can find/

Reply:
Updated

Reply:
Updated